css/lory.less

```
@@ -0,0 +1,49 @@
+.slider {
+  position: relative;
+  margin-top: 100px;
+}
+
+.frame {
+  position: relative;
+  // font-size: 0;
+  // line-height: 0;
+  overflow: hidden;
+  white-space: nowrap;
+
+  height: 650px;
+}
+
+.slides {

```

using LESS means you don't have to write CSS by repeating classes. You should clean up all your LESS files.

example:
```suggestion
.slides {
    @media screen and (max-width: 768px) {
          some media query styles for 768px...
    }
    some styles..
      li {
          some styles....
          
          &:hover {
                       hover styles....
          }
      }
}
```